independent-projects/bootstrap/app-model/src/main/java/io/quarkus/maven/dependency/GACT.java

```diff
@@ -8,59 +9,7 @@
     private static final long serialVersionUID = 2860156541775021365L;
 
     public static GACT fromString(String str) {
-        return new GACT(split(str, new String[4], str.length()));
-    }
-
-    protected static String[] split(String str, String[] parts, int fromIndex) {
```

Comment:
Maybe just keep this one and make it public? It's not great but I'd prefer it over creating `io/quarkus/util/ArtifactKeyUtil.java` for it. I think all the other flavors of the artifact API should go away eventually in favor of the `io.quarkus.maven.dependency` API.

Reply:
As you wish :)